src/datasets/summarizations/category-topk-coverage.summarization.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { queryFactory } from './category-topk-coverage.summarization';
+import { getSummaries, hasHighValidity, isTextPartsInSummary } from './utils/tests';
+import { CategoricalPoint } from '../metas/types';
+
+describe('queryFactory', () => {

```

Comment:
These tests don't test the coverage part of the summarization

Reply:
Fixed